lib/graphql/mutations.js

```diff
@@ -253,6 +253,7 @@ export const addEditCollectiveMutation = graphql(editCollectiveQuery, {
         'backgroundImage',
         'settings',
         'hostFeePercent',
+        'isDeletable',
```

Comment:
Why adding `isDeletable` here? Seems to me this is a problem?

Reply:
> Also, similar to collectives, we would like to prevent events with financial transactions to be deleted. If we're using deleteCollective, this might be already handled on the API side. On the frontend side, we would need to check the isDeletable property and remove/disable the delete button in the page.

Reply:
`export default addDeleteCollectiveMutation(addEditCollectiveMutation(EditEvent));`

Reply:
Is it makes sense?

Reply:
I don't see why we would add `isDeletable` to the mutation. It's not supported by the mutation, it is even breaking it. We've seen in the e2e tests.

Adding to `isDeletable` to the query https://github.com/opencollective/opencollective-frontend/pull/2996/files/a1e1c9bd25b7b658b309e2c03aa3e2ebba4f2e0f#r348933057 makes total sense, but in the mutation I don't see why.

Reply:
Yes, I understand. IsDeletable Removed.